ui/components/institutional/custody-confirm-link-modal/custody-confirm-link-modal.js

```diff
@@ -0,0 +1,137 @@
+import React, { useContext } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { MetaMetricsContext } from '../../../contexts/metametrics';
+import {
+  MetaMetricsEventName,
+  MetaMetricsEventCategory,
+} from '../../../../shared/constants/metametrics';
+import Button from '../../ui/button';
```

Comment:
Should be able to use the new Button from component-library

Reply:
Fixed, thanks